ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/pages/court-building-video/court-building-video.component.ts

```diff
@@ -17,7 +17,7 @@ export class CourtBuildingVideoComponent extends VideoViewBaseComponent {
   }
 
   continue() {
-    this.videoViewComponent.ngOnDestroy();
+    //this.videoViewComponent.ngOnDestroy();//
```

Comment:
This will cause an issue with Firefox

Reply:
It seems to working on firefox. I have been playing around. Just need to confirm on Safari as well. If it doesn't work, then I have to get AudioTracks and remove them